wd_core.au3

```diff
@@ -985,56 +985,63 @@ EndFunc   ;==>_WD_GetSource
 ; Parameters ....: $sSession - Session ID from _WD_CreateSession
 ;                  $sCommand - One of the following actions:
 ;                  |
-;                  |ADD    - Create a new cookie. $sOption has to be a JSON string
-;                  |DELETE - Delete a single cookie. The name of the cookie to delete is specified in $sOption
-;                  |GET    - Retrieve the value of a single cookie. The name of the cookie to retrieve has to be specified in $sOption
-;                  |GETALL - Retrieve the values of all cookies
+;                  |ADD       - Create a new cookie. $sOption has to be a JSON string
+;                  |DELETE    - Delete a single cookie. The name of the cookie to delete is specified in $sOption
+;                  |DELETEALL - Delete all cookies
+;                  |GET       - Retrieve the value of a single cookie. The name of the cookie to retrieve has to be specified in $sOption
+;                  |GETALL    - Retrieve the values of all cookies
 ;                  $sOption  - [optional] a string value. Default is ""
 ; Return values .: Success - Requested data returned by web driver.
 ;                  Failure - "" (empty string) and sets @error to one of the following values:
 ;                  - $_WD_ERROR_Exception
 ;                  - $_WD_ERROR_InvalidDataType
+;                  - $_WD_ERROR_InvalidArgue
 ; Author ........: Danp2
-; Modified ......:
-; Remarks .......: Please have a look at WD_Demo.au3 > DemoCookies function for how to add a new cookie
+; Modified ......: mLipok
+; Remarks .......: Please have a look at wd_demo.au3 > DemoCookies function for how to add a new cookie
 ; Related .......:
 ; Link ..........: https://www.w3.org/TR/webdriver#cookies
 ; Example .......: No
 ; ===============================================================================================================================
 Func _WD_Cookies($sSession, $sCommand, $sOption = Default)
 	Local Const $sFuncName = "_WD_Cookies"
 	Local $sResult, $sResponse, $iErr
-
 	If $sOption = Default Then $sOption = ''
 
-	Switch $sCommand
-		Case 'add'
-			$sResponse = __WD_Post($_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession & "/cookie", $sOption)
-			$iErr = @error
+	$_WD_ERROR_InvalidArgue

```

Comment:
This looks like a copy / paste issue

Reply:
Fixed: [$_WD_ERROR_InvalidArgue fixed](https://github.com/Danp2/au3WebDriver/pull/195/commits/0651d63a117874ed83c3a4e574d95f308a4d964e)